x-pack/plugins/secops/public/components/visualization_placeholder.tsx

```diff
@@ -26,27 +29,52 @@ export const VisualizationPlaceholder = styled(EuiPanel)`
   }
 `;
 
+export const ProviderContainer = styled.div`
+  margin: 5px;
+  user-select: none;
+  cursor: grab;
+`;
+
 interface Props {
+  timelineId: string;
   count: number;
   myRoute: string;
+  dispatch: Dispatch;
 }
 
 /** TODO: delete this stub */
-export const Placeholders = pure<Props>(({ count, myRoute }) => (
-  <React.Fragment>
-    {range(0, count).map(p => (
-      <VisualizationPlaceholder
-        data-test-subj="visualizationPlaceholder"
-        key={`visualizationPlaceholder-${p}`}
-      >
-        <WhoAmI data-test-subj="whoAmI" sourceId="default">
-          {({ appName }) => (
-            <div>
-              {appName} {myRoute}
-            </div>
-          )}
-        </WhoAmI>
-      </VisualizationPlaceholder>
-    ))}
-  </React.Fragment>
-));
+class PlaceholdersComponent extends React.PureComponent<Props> {
+  public render() {
+    const { count, dispatch, myRoute, timelineId } = this.props;
+
+    return (
+      <React.Fragment>
+        {range(0, count).map(i => (
+          <VisualizationPlaceholder
+            data-test-subj="visualizationPlaceholder"
+            key={`visualizationPlaceholder-${i}`}
+          >
+            <WhoAmI data-test-subj="whoAmI" sourceId="default">
+              {({ appName }) => (
+                <div>
+                  {appName} {myRoute}
+                </div>
+              )}
+            </WhoAmI>
+            <ProviderContainer
+              onClick={e => {
```

Comment:
On every PR I encourage fully spelled out names such as `event`, but no changes required. I just always do. Like why not?

Also, I don't think you use the `event` here do you? You can just do `onClick={() =>`

Reply:
Also, per some convention if the arg isn't captured for any specific purpose, rather than passing an empty arg context, in such case that you don't know the use of this for future, pass a `_` as an explicit and declarative point in the right direction.
A matter of preference, just a second opinion from a random guy.